### PR TITLE
Add "How to install" to README

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,29 @@ codechain-keystore-cli [![Build Status](https://travis-ci.org/CodeChain-io/codec
 
 A command line CodeChain key management tool
 
+
+How to install
+----------------
+
+### Binary install
+
+You can download recent binaries from [here](https://github.com/CodeChain-io/codechain-keystore-cli/releases/tag/0.1.1)
```
Comment (kseo) [inline README.md]: recent -> the latest

File: README.md [hunk 2]
```diff
@@ -3,6 +3,29 @@ codechain-keystore-cli [![Build Status](https://travis-ci.org/CodeChain-io/codec
...
+
+### Install using a package manager(npm)
```
Comment (kseo) [inline README.md]: Add a space before (npm)

File: README.md [hunk 3]
```diff
@@ -3,6 +3,29 @@ codechain-keystore-cli [![Build Status](https://travis-ci.org/CodeChain-io/codec
...
+
+```sh
+npm install -g codechain-sdk-cli
+```
+
+### Install using a package manager(yarn)
```
Comment (kseo) [inline README.md]: Ditto

**Comment by majecty** [comment]
@kseo I fixed it

**Pull request merged**